Gopkg.toml

```diff
@@ -23,7 +23,7 @@
 
 [[constraint]]
   name = "cloud.google.com/go"
-  version = "0.16.0"
+  version = "0.19.0"
```

Comment:
Is the version upgrade related to fixing leaks? If so, what's the upstream fix/issue?

If not related to fixing the leaks, would be better to use a separate commit.

Reply:
True. It was rather my attempt for fixing https://github.com/improbable-eng/thanos/issues/234 ;p but seems that new version has leak as well (: Can revert this.